doc: Update migration docs post TTN V2 shutdown (#676)
TheThingsIndustries/lorawan-stack-docs: doc/content/getting-started/migrating/migrating-from-v2/_index.md

<!--more-->

> {{< note >}} The Things Network {{% ttnv2 %}} machines have been completely shut down on **December 8, 2021**, so it is not possible to migrate devices from The Things Network {{% ttnv2 %}} to {{% tts %}} anymore. See [Adding Devices]({{< ref "/devices/adding-devices" >}}) section to start adding devices to {{% tts %}} from scratch.

Reviewer: This shouldn't be a note

```suggestion
{{< note >}} The Things Network {{% ttnv2 %}} machines have been completely shut down on **December 8, 2021**, so it is not possible to migrate devices from The Things Network {{% ttnv2 %}} to {{% tts %}} anymore. See [Adding Devices]({{< ref "/devices/adding-devices" >}}) section to start adding devices to {{% tts %}} from scratch.
```

> On the other hand, private {{% ttnv2 %}} deployments might still be running. This section can guide you through the process of migrating devices from private {{% ttnv2 %}} deployments to {{% tts %}}.

Reviewer: ```suggestion
If your private {{% ttnv2 %}} deployment is still running, this section can guide you through the process of migrating devices from private {{% ttnv2 %}} deployments to {{% tts %}}.
```

<!--more-->

> <<<<<<< HEAD

Reviewer 2: Merge conflict.
